sqlg-postgres-parent/sqlg-postgres-dialect/src/main/java/org/umlg/sqlg/sql/dialect/PostgresDialect.java

```diff
@@ -3301,6 +3304,20 @@ public String getFullTextQueryText(FullText fullText, String column) {
         return "to_tsvector('" + fullText.getConfiguration() + "', " + leftHand + ") @@ " + toQuery + "('" + fullText.getConfiguration() + "',?)";
     }
 
+    @Override
+    public String getArrayContainsQueryText(String column, ArrayContains<?> predicate) {
+        List<String> predicateValues =
```

Comment:
Is it possible to have a string array?  If it is, would you need to include the quotes in the predicate values?

Reply:
That is a good point. Instead of building entire query with values here, I will use `?` as place holder to be replaced by actual value in the statement later. The sqlg library supports many types, so better to re-use the existing conversion mechanism instead of writing one again.